### PR TITLE
Fix syntax highlighting and add files key to package.json

### LINKED ISSUE #13: Use JavaScript hilighting instead of language generic
In `README.md`. You can view [the readme on my fork](https://github.com/dawsonbotsford/prototypes) to see what this looks like.

### PR DESCRIPTION
**Pull request by dawsonbotsford**
Closes #13 and #14

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,6 +6,10 @@
 	"contributors": ["Alex Fernández <alexfernandeznpm@gmail.com>"],
 	"license": "MIT",
 	"main": "index.js",
+  "files": [
```
Comment (alexfernandez) [inline package.json]: Please indent with tabs.

**Comment by alexfernandez** [comment]
Good job, just needs a couple of tabs.

**Comment by dawsonbotsford** [comment]
Thanks, I'll work on getting my config setup and fixing those tabs. Are you alright with adding a `.editorconfig` to the directory to help others in the future? http://editorconfig.org/

**Comment by alexfernandez** [comment]
An `.editorconfig` sounds great!

**Comment by dawsonbotsford** [comment]
These diffs are now very very ugly, apologies. That's what happens when you have issues with space vs. tabs though! All updates, you should give it a test, I think you'll like the little changes I just pushed on the last one (like adding `jshint` to the dev deps)

**Comment by alexfernandez** [comment]
Now it's failing for Node.js v0.8 because the version of JSHint is not compatible. Can you just change the dependency to `"*"` so it works? Not many people are using v0.8, I guess, but there is no reason to bother those unlucky few.

**Comment by dawsonbotsford** [comment]
Yikes, I looked into that failing build and it looks like jshint fails the install on versions of npm as old as you are testing. They did patch it in May, 2014, so if you updated your npm version, you wouldn't need to worry about it. 

Switching to version `*` doesn't help since it was already failing on the newest build. You can read more about it [here](https://github.com/npm/npm/issues/5298) if you'd like.

**Comment by alexfernandez** [comment]
Pardon my ignorance: how can I update npm on Travis-CI?

**Comment by dawsonbotsford** [comment]
np, all you have to do is change the version you cover in your `.travis.yml`. I think my most recent push may just trigger that anyways. So now you'd just have to merge it (if all tests pass now). It's a waiting game...

**Comment by dawsonbotsford** [comment]
I reverted the entire jshint part. To keep it simple and pass all tests.

**Comment by alexfernandez** [comment]
Done, thanks!

**Comment by dawsonbotsford** [comment]
Thanks you too! Sorry for all of that back-and-forth, that wasn't supposed to happen

**Comment by alexfernandez** [comment]
No problem, especially if you answer this fast :)

**Pull request merged**